backend/model/notifications.js

```diff
@@ -0,0 +1,19 @@
+module.exports = (sequelize, Seq) => {
+	return sequelize.define('notification', {
+		id : {
+			type : Seq.INTEGER,
+			primaryKey: true,
+			autoIncrement: true,
+		},
+		userId : Seq.INTEGER,
+		type : Seq.TEXT,
+		isRead : Seq.TINYINT,
+		title : Seq.TEXT,
+		body : Seq.TEXT,
+		link : Seq.TEXT,
+		other : Seq.TEXT
+	}, 
+	{
+		timestamps : true
+	})
+}
```

Comment:
can you remove notification.js file? Because fields of this model are gonna probably change. And it becomes hard to change model's fields once the table created on server database.

Reply:
@hazerb could apply these changes? I have problems accessing these branches.

Reply:
Okay i applied changes.